modules/transport/PostMessageTransportBackend.js

```diff
@@ -0,0 +1,174 @@
+import Postis from 'postis';
+
+/**
+ * The default options for postis.
+ *
+ * @type {Object}
+ */
+const DEFAULT_POSTIS_OPTIONS = {
+    window: window.opener || window.parent
+};
+
+/**
+ * The list of methods of incoming postis messages that we have to support for
+ * backward compatibility for the users that are directly sending messages to
+ * Jitsi Meet (without using external_api.js)
+ *
+ * @type {string[]}
+ */
+const LEGACY_INCOMING_METHODS = [
+    'avatar-url',
+    'display-name',
+    'email',
+    'toggle-audio',
+    'toggle-chat',
+    'toggle-contact-list',
+    'toggle-film-strip',
+    'toggle-share-screen',
+    'toggle-video',
+    'video-hangup'
+];
+
+/**
+ * The list of methods of outgoing postis messages that we have to support for
+ * backward compatibility for the users that are directly listening to the
+ * postis messages send by Jitsi Meet(without using external_api.js).
+ *
+ * @type {string[]}
+ */
+const LEGACY_OUTGOING_METHODS = [
```

Comment:
> But also this change is from @lyubomir 's commit so maybe he will have other arguments as well.

I don't know what these are. My change was in the format of the name i.e. I changed it from `legacyOutgoingMethods` to `LEGACY_OUTGOING_METHODS` in order to reflect the fact that these are hard-coded constants. Internally, these are methods in the terms of postis. Externally, they're what? Message names? I'm not really proficient enough in the naming/terminology of this PR.